### DISCUSSION TITLE
How to install SSL certificate on clouds tack 4.19 version

### DISCUSSION THREAD

**Discussion by titu044** [apache/cloudstack#9149] [Q&A] [answered]
This is not working anyone help me 
I have wildcard ssl certificate when i convert PEM to JKS and Upload my management server /etc/cloudstack/management/server.properties 

https.enable=true
https.port=8443
https.keystore=/etc/cloudstack/management/cloud.jks
https.keystore.password=vmops.com

**Comment by pcfriek1987** [comment]
does it work with a jks? I'm using a pkcs12.

**Comment by yadvr** [comment, marked as answer]
Follow https://www.shapeblue.com/securing-cloudstack-4-11-with-https-tls/
